qiita_ware/ebi.py

```diff
@@ -981,3 +950,52 @@ def send_xml(self):
             print "FAILED"
 
         return (study_accession, submission_accession)
+
+    def generate_demultiplexed_fastq(self):
+        """Generates demultiplexed fastq
+
+        Returns
+        -------
+        demux_samples
+            List of successful demultiplexed samples
+
+        Notes
+        -----
+        - As a performace feature, this method will check if self.ebi_dir
```

Comment:
If this fails the code will remove the folder so I really do not see how that can happen. Anyway, if you feel strongly about this, I can put a parameter 'regenerate' by default FALSE ...